src/components/DistanceRequest.js

```diff
@@ -34,8 +37,20 @@ const MAP_PADDING = 50;
 const DEFAULT_ZOOM_LEVEL = 10;
 
 const propTypes = {
-    /** The transactionID of this request */
-    transactionID: PropTypes.string,
+    /** Holds data related to Money Request view state, rather than the underlying Money Request data. */
+    iou: PropTypes.shape({
+        id: PropTypes.string,
+        amount: PropTypes.number,
+        currency: PropTypes.string,
+        participants: PropTypes.arrayOf(participantPropTypes),
+        transactionID: PropTypes.string,
+    }),
+
+    /** Type of money request (i.e. IOU) */
+    iouType: PropTypes.string,
```

Comment:
This should be updated to something like an enum. I think it can be based on `CONST.IOU.MONEY_REQUEST_TYPE`
```suggestion
    iouType: PropTypes.oneOf(_.values(CONST.IOU.MONEY_REQUEST_TYPE)),
```